cmd/dep/status.go

```diff
@@ -222,9 +222,9 @@ func (cmd *statusCommand) Run(ctx *dep.Ctx, args []string) error {
 
 	if digestMismatch {
 		if hasMissingPkgs {
-			ctx.Loggers.Err.Println("Lock inputs-digest mismatch due to the following packages missing from the lock:\n")
+			ctx.Loggers.Err.Printf("Lock inputs-digest mismatch due to the following packages missing from the lock:\n\n")
 			ctx.Loggers.Out.Print(buf.String())
-			ctx.Loggers.Err.Println("\nThis happens when a new import is added. Run `dep ensure` to install the missing packages.")
+			ctx.Loggers.Err.Printf("\nThis happens when a new import is added. Run `dep ensure` to install the missing packages.")
```

Comment:
I tested it last night both ways and strangely, it didn't change what was printed to the console which made my head hurt. 😀  I've pushed and squashed adding it back though, because I'm sure I'm just missing something.